packages/edit-site/src/components/header/document-actions/style.scss

```diff
@@ -0,0 +1,14 @@
+.edit-site-document-actions {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: space-evenly;
+	margin: 5px 0;
```

Comment:
I'm not a huge fan of how this margin is different from the rest of the items in the header, but it does match the latest in #25085 🤔